app/config/config.yml

```diff
@@ -345,3 +345,6 @@ old_sound_rabbit_mq:
             queue_options:
                 name: 'wallabag.import.chrome'
             callback: wallabag_import.consumer.amqp.chrome
+
+fos_js_routing:
+    routes_to_expose: [ homepage, starred, archive, all, tag, config, import, developer, howto, about, logout ]
```

Comment:
Can't you explose the array in a list instead? It'll be more easy to read the diff in the future.

```yaml
    routes_to_expose:
        - homepage
        - starred
```